client/src/style.css

```diff
@@ -1274,8 +1274,21 @@ sandbox header
     opacity: 0;
     transition: opacity 0.5s 0.25s; /* 👈 Change the time to meet your requirements. */;
     z-index: 1000;
+    width: max-content;
+    max-width: 200px;
+    white-space: wrap;
```

Comment:
suggest adding  `pointer-events: none;` here, so that the tooltip does not block mouse clicks on termsetting pills or other inputs that may be blocked by the tooltip text.

Reply:
Added but it does not make it disappear when hovering over nearby elements

Reply:
The tooltip disappears for me, but I also had to restore the css style to not use the fadeout:

https://github.com/user-attachments/assets/4277678b-d5cb-46ce-9687-b5fa450b9e21`

```css
.sja_root_holder [aria-label]:after, .sja_menu_div [aria-label]:after, .sja_pane [aria-label]:after {
    content: attr(aria-label);
    position: absolute;
    left: 1em;
    top: 1.25em; /* put it on the top */
    /*bottom: 50%;*/
    background-color: #aaa;
    border-radius: 6px ;
    font-size: 15px;
    color: #fff;
    padding: 5px;
    opacity: 0;
    transition: opacity 0.5s 0.25s; /* 👈 Change the time to meet your requirements. */;
    z-index: 1000;
    width: max-content;
    max-width: 200px;
    white-space: wrap;
    pointer-events: none;
  }

  .sja_root_holder [aria-label]:hover:after, .sja_menu_div [aria-label]:hover:after, .sja_pane [aria-label]:hover:after {
    opacity: 1;
  }
```